pkgs/development/tools/gron/default.nix

```diff
@@ -2,7 +2,7 @@
 
 buildGoPackage rec {
```

Comment:
Can we use buildGoModule here?

Reply:
I tried and it has some dependency problem, so since it's a minor version and the declared dependencies are exactly the same as the previous version I just bumped the version for now.